### DISCUSSION TITLE
texstudio compile  报错

### DISCUSSION THREAD

**Discussion by ujust4fun** [sjtug/SJTUThesis#911] [提问] [answered]
![image](https://user-images.githubusercontent.com/25298124/229326627-b70f6d03-89b8-4af0-84c8-5a766b3b97a7.png)

脚本编译是可以的，但是texstudio就报错了，不明白该怎么修改texstudio的配置

**Comment by note286** [comment]
你在终端里执行`xelatex main`试试。

**Reply by ujust4fun** [reply]
谢谢，重启了一下texstudio就解决了

**Comment by AlexaraWu** [comment, marked as answer]
Dupe #903

请更新至最新的 TeX 发行版。

**Comment by ujust4fun** [comment]
好滴。还有一个问题。
模板修改def文件会报错呢，难道是要从sjtutex这个包里重新编译吗。我想照着学校的模板改几个字。比如说下图中的“Doctor->Doctoral”

![image](https://user-images.githubusercontent.com/25298124/229328656-8e1d860b-6f95-4917-b6b3-307226546998.png)

**Reply by AlexaraWu** [reply]
`setup.tex` 里重新设置 `info/en/subject` 就行了。

**Comment by ujust4fun** [comment]
infor中的字段信息去哪里可以手动添加呢？
我想把首页的一些中文字段更改名称；
以及第二页的时间这个格式2的角标放下来
![image](https://user-images.githubusercontent.com/25298124/229329588-ed576284-f621-4e71-a234-e11d51dadedb.png)

**Reply by ujust4fun** [reply]
好像没有表达清楚。我知道在setup中info有字段，但是不知道去哪里找到全部的字段信息以及自定义

**Reply by AlexaraWu** [reply]
可供修改的 `info` 字段可以参考 sjtutex.pdf 文档，其他字段不支持修改。

**Reply by ujust4fun** [reply]
OK，谢谢！😉

**Comment by ujust4fun** [comment]
又遇到了一个问题。info中的日期data字段失效了
![Uploading image.png…]()

**Reply by ujust4fun** [reply]
![image](https://user-images.githubusercontent.com/25298124/229333758-d45785c1-971c-4125-b6fe-bd31c3aad98d.png)

**Reply by LogCreative** [reply]
可能你上一个键值对之后没加逗号

**Reply by ujust4fun** [reply]
都加了的😶

**Reply by ujust4fun** [reply]
![image](https://user-images.githubusercontent.com/25298124/229334360-8a75f3a1-c24e-4ebb-babe-9ee28e861d82.png)

**Reply by ujust4fun** [reply]
好了我解决了。

en/subject后不能有逗号

**Reply by LogCreative** [reply]
是你有空行导致的 前面有注释说明

**Comment by ujust4fun** [comment]
问一下，首页那个logo能大一点吗🤣

**Reply by LogCreative** [reply]
导言区使用

```latex
\EditInstance{sjtu}{title/zh/logo}{content={\includegraphics[width=4cm]{sjtu-vi-badge-red.pdf}}}
```

**Reply by ujust4fun** [reply]
谢谢！
